CMakeLists.txt

```diff
@@ -132,6 +132,13 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
         clr_unknown_arch()
     endif()
     set(CLR_CMAKE_PLATFORM_LINUX 1)
+
+    # Detect Alpine Linux
+    file(READ "/etc/os-release" OS_RELEASE)
```

Comment:
Good point. I have assumed file read just returns an empty variable, but it actually fails instead. So I am adding a check for the file existence.